wire-ios-request-strategy/Sources/Message Sending/MessageSenderTests.swift

```diff
@@ -0,0 +1,356 @@
+////
```

Comment:
I forgot this was one of the framework all tests sit in same directory of source code for some reason

Reply:
Heh, someone was really keen on that in past maybe not the best decision in hindsight